src/lib/pages/backoffice/Patron/PatronDetails/PatronCurrentBorrowingRequests/PatronCurrentBorrowingRequests.test.js

```diff
@@ -6,6 +6,7 @@ import { mount, shallow } from 'enzyme';
 import React from 'react';
 import { BrowserRouter } from 'react-router-dom';
 import PatronCurrentBorrowingRequests from './PatronCurrentBorrowingRequests';
+import 'jest-canvas-mock';
```

Comment:
If we add the `truncate` prop to `LiteratureTitle` we will render the `Truncate` component from the [React-Truncate](https://www.npmjs.com/package/react-truncate) and it requires `canvas`. Since we have tests for the `PatronCurrentBorrowingRequests` if we add the truncate it will fail and throw something like this `Not implemented: HTMLCanvasElement.prototype.getContext (without installing the canvas npm package)`. And the easiest solution I found is to use [jest-canvas-mock](https://github.com/hustcc/jest-canvas-mock) _Mock canvas when run unit test cases with jest._

Reply:
Thanks a lot for the clear explanation!
Can you maybe add a comment here? Then it is ready to go! 💯